src/schema/objects/metadata.yaml

```
@@ -561,6 +561,53 @@ DecayCorrectionFactor:
   type: array
   items:
     type: number
+DeidentificationMethod:
+  name: DeidentificationMethod
+  display_name: Deidentification Method
+  description: |
+    A description of the mechanism or method used to remove the Patient's identity.
+    Corresponds to DICOM Tag 0012, 0063 `De-identification Method`.
+  type: string
```

Following up on https://github.com/rordenlab/dcm2niix/pull/813#discussion_r1674566934.

I'm not sure if this practically shows up in DICOM as a list of strings, or if it is either a string or a list of strings.

If we want to generally make this an array of strings, it would be:

```suggestion
  type: array
  items:
    type: string
```

If it would be a string or an array of strings:

```suggestion
  anyOf:
    - type: string
    - type: array
      items:
        type: string
```

I tend to prefer the simpler array of strings, rather than allowing it to sometimes be a string, but I'm okay following what DICOM does, for consistency.

---

Would be happy to approve this and get it merged once the type is updated per @effigies suggestion.

---

I am at a conference this week. I can make the change next week when I return.
Thank you both for your review!